gnmpagerduty/__init__.py

```diff
@@ -0,0 +1,8 @@
+from plugin import *
+from tasks import check_storage
+
+__version__ = 'DISPLAY_VERSION'
+__production__ = False
+
+check_storage.delay()
```

Comment:
I don;t thinks that this is a good idea.  Use the crontab function decorator from Celery to call the task regularly via celerybeat; this approach will cause problems with lots of duplicate tasks when run in the clustered environment.